docker-compose.linux-dev.yaml

```diff
@@ -39,7 +26,7 @@ services:
     volumes:
       - ./python:/python:cached
       - ./data:/data:cached
-    extra_hosts:
-      - "host.docker.internal:host-gateway"
     environment:
       - EXPERIMENT_ID=e52b39624588791a7889e39c617f669e
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
```

Comment:
why is this added now?

Reply:
Oh Iva I didn't get notifications for this or I'd have answered much sooner. This needs to be added to the linux yaml file for the worker to function. I don't exactly know why, but Marcell figured it out and it solved the problems we were having in Linux two months ago

Reply:
I see, cool, I was making sure it is not a copy-pasta out of mistake